### ISSUE TITLE
Schema aggregator: $table->datetimes() not handled

### ISSUE THREAD

**Issue by alies-dev** [psalm/laravel-psalm-plugin#529]
## Description

`Blueprint::datetimes()` creates nullable `created_at` and `updated_at` columns (datetime type), but the schema aggregator silently ignores it.

`datetimes()` is not in `METHODS_USE_HARDCODED_COLUMN_NAME` (which handles `timestamps`, `timestampstz`, etc.), so when called without arguments it hits the `continue` at line 446 ("unknown type [method call without args]") and the columns are never registered.

## Laravel source (present in both Laravel 12 and 13)

```php
// Blueprint.php
public function datetimes($precision = null)
{
    $this->datetime('created_at', $precision)->nullable();
    $this->datetime('updated_at', $precision)->nullable();
}
```

## Impact

Migrations using `$table->datetimes()` instead of `$table->timestamps()` will not have `created_at` / `updated_at` recognized by the schema aggregator, causing false positives on property access.

## Suggested fix

Add `'datetimes'` to `METHODS_USE_HARDCODED_COLUMN_NAME` and handle it in the no-args switch (lines 416-438) alongside the existing `timestamps` case:

```php
case 'datetimes':
case 'timestamps':
case 'timestampstz':
case 'nullabletimestamps':
case 'nullabletimestampstz':
    $table->setColumn(new SchemaColumn('created_at', 'string', true));
    $table->setColumn(new SchemaColumn('updated_at', 'string', true));
    break;
```
